docs/pact-properties.md

```diff
@@ -392,13 +392,13 @@ table.
   ("Transfer money between accounts"
     (properties [(row-enforced 'accounts 'ks from)]))
 
-  (let ((from-bal (at 'balance (read 'accounts from)))
```

Comment:
But Stu prefers these to be `'quoted`

Reply:
When I use the non-quoted version, this typechecks. When I use the quoted version, I get

```
examples/demo/accounts.pact:26:33:Warning: Unable to resolve type (object:<{at5_o}>)
examples/demo/accounts.pact:26:33:Warning: Unable to solve overloaded function: read

examples/demo/accounts.pact:26:33:Warning: Unable to resolve type (object:<{at5_o}>)
examples/demo/accounts.pact:26:33:Warning: Unable to solve overloaded function: read
```

Reply:
My mistake here -- I got mixed up.